src/containers/ClassifierTester.jsx

```diff
@@ -0,0 +1,100 @@
+/*
+Classifier Tester
+-----------------
+
+This component is used to test our ability to connect to the Panoptes API
+service, without having to worry about the presentational side of things.
+
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchProject, PROJECT_STATUS } from '../ducks/project';
+
+class ClassifierTester extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <main className="app-content tester-page">
+        <div className="control-panel">
+          <button className="green button" onClick={this.fetchProject.bind(this)}>Fetch Project</button>
```

Comment:
can this bind be put in the constructor? `this.fetchProject = this.fetchProject.bind(this)`